Makefile

```diff
@@ -17,6 +17,9 @@ COORDINATOR_ARGS=-mod=readonly -trimpath ./cmd/ltcoordinator
 AGENT=ltagent
 AGENT_ARGS=-mod=readonly -trimpath ./cmd/ltagent
 
+export GOBIN = $(PWD)/bin
+export PATH = $(shell printenv PATH):$(GOBIN)
```

Comment:
Is there a reason we are not doing just:

```sh
PATH=$PATH:$GOBIN
```

Reply:
Yes, we need to do "shell printenv" because otherwise this happens- Recursive variable `PATH' references itself (eventually). .

And I needed to do export because without that it's just a simple variable set, and sub-processes spawned from the Makefile command do not get them. But somehow they do now. Will change.